docs/_static/network/v1.yaml

```diff
@@ -48,6 +50,8 @@ paths:
                 type: string
         "404":
           description: "Transaction wasn't found in the transaction log"
+        default:
+          $ref: '../common/error_response.yaml'
```

Comment:
default is for all 400+ status codes. In other apis the description for error returns is documented per path under path.get.description